packages/terra-arrange/examples/index.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
-import PropsTable from '../../../packages/terra-site/src/PropsTable';
-import Markdown from '../../../packages/terra-site/src/Markdown';
+import PropsTable from '../../../packages/terra-props-table/src/PropsTable';
+import Markdown from '../../../packages/terra-markdown/src/Markdown';
```

Comment:
@Matt-Butler  Thank You for pointing me in the right direction.
@Matt-Butler @bjankord Updated the code. :)